plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/tasks/GenerateXmlTask.groovy

```diff
@@ -20,8 +29,14 @@ class GenerateXmlTask extends AbstractDslTask {
     @Override
     Map<String, ?> getProperties() {
         [
-                outputDirectory: "${project.buildDir}/jobdsl/xml"
+                outputDirectory    : "${project.buildDir}/jobdsl/xml",
+                failOnMissingPlugin: failOnMissingPlugin,
         ]
     }
 
+    @Option(option = 'failOnMissingPlugin', description = 'Fail the task if the plugin is missing.')
```

Comment:
'Fail the task if a required plugin is missing.'

Reply:
Done